src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -731,6 +745,39 @@ public function links()
     }
 
     /**
+     * Returns an Image object for the first node in the list.
+     *
+     * @return Image An Image instance
+     *
+     * @throws \InvalidArgumentException If the current node list is empty
+     */
+    public function image()
+    {
+        if (!count($this)) {
+            throw new \InvalidArgumentException('The current node list is empty.');
+        }
+
+        $node = $this->getNode(0);
```

Comment:
We should make sure `$node` is an instance of `\DOMElement`, just like we do in the `link()` method.